editoast/src/views/infra/mod.rs

```diff
@@ -161,6 +161,7 @@ async fn rename(
     let new_name = new_name?.0.name;
     conn.run(move |conn| {
         let infra = Infra::rename(conn, infra, new_name)?;
+        let infra = Infra::updated_metadata(&infra, conn)?;
```

Comment:
Should we update the `modified` timestamp when we rename an infra?

In my opinion we should update it only when the railjson change. Otherwise we could discuss why it's not updated when we lock or unlock it.